client/components/main/index.jsx

```diff
@@ -11,7 +11,11 @@ module.exports = React.createClass( {
 
 	render: function() {
 		return (
-			<main className={ classnames( this.props.className, 'main' ) } role="main">
+			<main
+				className={ classnames( this.props.className, 'main' ) }
+				role="main"
+				style={ this.props.style }
+			>
 				{ this.props.children }
 			</main>
```

Comment:
I personally don't mind `this.props` especially if there aren't too many items being called.